### ISSUE TITLE
Failed to download 'zalenium-github/zalenium'

### ISSUE THREAD

**Issue by boxcee** [zalando/zalenium#1023]
## 🐛 Bug Report

Cannot install helm chart. Seems like it was removed.

```
helm install --name my-release --namespace my-zalenium zalenium-github/zalenium
Error: failed to download "zalenium-github/zalenium" (hint: running `helm repo update` may help)
```

## To Reproduce

```
helm repo add zalenium-github https://raw.githubusercontent.com/zalando/zalenium/master/charts/zalenium
helm install --name my-release --namespace my-zalenium zalenium-github/zalenium
```

## Expected behavior

```
helm search zalenium-github -l

NAME                    	CHART VERSION	APP VERSION	DESCRIPTION            
zalenium-github/zalenium	3.141.59     	3.141.59   	Chart for selenium grid
```
So I would expect the chart to be downloadable.

**Comment by boxcee** [comment]
Whatever #1020 did, it broke something else.

**Comment by diemol** [comment]
@ramapalani could you please have a look? 
Otherwise I'll probably need to revert #1020

**Comment by diemol** [comment]
Closed by mistake, sorry!

**Comment by sharkymcdongles** [comment]
I fixed this in my pull request: https://github.com/zalando/zalenium/pull/1025

@diemol @ramapalani

**Comment by jb-brossard** [comment]
Still having the issue

**Comment by diemol** [comment]
Fixed via #1025, thanks @sharkymcdongles!
